recipes/sqlitecpp/all/test_package/CMakeLists.txt

```
@@ -1,10 +1,8 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.4)
```

Because CXX_STANDARD is available since 3.4